### PR TITLE
Feature/endpoints

### PR DESCRIPTION
**Pull request by andres-cox**
Added new endpoints. could you verify my new files.

### CODE REVIEW

File: code_editor/urls.py [hunk 1]
```diff
@@ -1,6 +1,24 @@
+#
+# @main.py Copyright (c) 2020 Jalasoft.
```
Comment (alvaro-cruz) [inline code_editor/urls.py]: Must be @url.py

File: jala_compiler/urls.py [hunk 2]
```diff
@@ -36,4 +36,3 @@
             'document_root': settings.MEDIA_ROOT,
         }),
     ]
-
```
Comment (alvaro-cruz) [inline jala_compiler/urls.py:39]: Leave one empty extra line

File: code_editor/views.py [hunk 3]
```diff
@@ -1,22 +0,0 @@
...
-
-    def post(self, request):
-        params = Parameters()
-        params.set_language('python')
-        params.set_file(settings.BASE_DIR / r'media\python\client_files\client_file.py')
-
-        program = Compiler(params)
-        output = program.execute()
-
-        return render(request, self.template_name, {'output': output})
```
Comment (alvaro-cruz) [inline code_editor/views.py:22]: Leave one empty extra line

File: code_editor/urls.py [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+
 
 urlpatterns = [
-    path('', CodeEditorView.as_view(), name='Demo'),
-]
\ No newline at end of file
+    path('file/', FileView.as_view(), name='file-view'),
+    path('file/<int:id>', FileEditView.as_view(), name='file-edit-view'),
+    path('project/', ProjectView.as_view(), name='project-view'),
+]
```
Comment (alvaro-cruz) [inline code_editor/urls.py:24]: Extra Line

**Review by alvaro-cruz — COMMENTED** [review]
Looks Good, but you need to put one extra line at the end of every file

**Comment by andres-cox** [comment]
I'll fix urls.py file, thanks

**Pull request closed without merging**